include/IMeshBuffer.h

```diff
@@ -157,6 +168,7 @@ GL_INVALID_OPERATION is generated by glVertexAttribPointer if size is GL_BGRA an
             size_t attrOffset[EVAI_COUNT];
             uint32_t attrDivisor[EVAI_COUNT];
 	public:
```

Comment:
Should add a comment with a link to https://www.khronos.org/opengl/wiki/Vertex_Specification
telling people to read that to understand.